src/main/java/com/hillel/items_exchange/config/SwaggerConfig.java

```diff
@@ -20,6 +26,28 @@ public Docket api() {
                 .select()
                 .apis(RequestHandlerSelectors.any())
                 .paths(PathSelectors.any())
+                .build()
+                .securitySchemes(List.of(apiKey()))
+                .securityContexts(List.of(securityContext()));
+    }
+
+    private ApiKey apiKey() {
+        return new ApiKey("JWT", "Authorization", "header");
+    }
+
+    private SecurityContext securityContext() {
+        return SecurityContext.builder()
+                .securityReferences(defaultAuth())
+                .forPaths(PathSelectors.any())
                 .build();
     }
+
+    List<SecurityReference> defaultAuth() {
```

Comment:
Why does 'package' access modifier here for?

Reply:
my bad, access can be "private". i change this in my next commit

Reply:
Done in commit "caaf656"